fdk/context.py

```diff
@@ -201,8 +203,11 @@ def is_sampled(self):
     def flags(self):
         return self.__flags
 
-    # this is a helper method specific for py_zipkin
     def zipkin_attrs(self):
+        return self.__zipkin_attrs
```

Comment:
I would add a Python doc string (in fact, we should probably have doc strings on all public methods), in this case specifying that this is a convenience method which returns a zipkin attributes named tuple in the format expected by py_zipkin only.

Reply:
Yep, I will go through the public methods after this change and add docstrings